### PR TITLE
[netapp-exporter] migrate alerts from capacity exporter to harvester

### PR DESCRIPTION
**Pull request by chuan137**

### CODE REVIEW

File: prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/alerts/storage/cinder.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+      - alert: VVolDatastoreFlexvolHighUsage
+        expr: netapp_volume_size_used{app="netapp-harvest-exporter-cinder", volume=~"vv.*"} / 1024/1024/1024 > 7000 and netapp_volume_size_used_percent{app="netapp-harvest-exporter-cinder", volume=~"vv.*"} > 75
+        for: 3h
+        labels:
+          severity: warning
+          tier: os
+          service: cinder
+          support_group: compute-storage-api
+          meta: 'VVol flex-volume usage over 90 percent of 5 TB or above 75 percent usage'
```
Comment (Carthaca) [inline prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/alerts/storage/cinder.yaml:42]:
```suggestion
          meta: 'VVol flex-volume usage usage larger than 7TB and above 75 percent usage'
```

File: prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/alerts/storage/cinder.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          playbook: 'docs/support/playbook/cinder/balancing/cinder_balance_alert'
+        annotations:
+          description: 'Usage of VVol flex-volume {{$labels.volume}} on filer {{$labels.filer}} in az {{$labels.availability_zone}} is high'
+          summary: 'Flex-volume {{$labels.filer}}/{{$labels.volume}} usage larger than 7TB or above 75 percent usage'
```
Comment (Carthaca) [inline prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/alerts/storage/cinder.yaml:46]:
```suggestion
          summary: 'Flex-volume {{$labels.filer}}/{{$labels.volume}} usage larger than 7TB and above 75 percent usage'
```

**Review by Carthaca — CHANGES_REQUESTED** [review]
`NetappFilerExporterAuthenticationError`, `NetappFilerExporterDNSError` and `NetappFilerExporterUnknownError` are removed because no longer relevant, right?

And some meta/summary of the vvol alerts had typos. They were there before, but a good chance to fix this on the way, I think

File: prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/alerts/storage/cinder.yaml (hunk 1 above)
Comment (chuan137) [reply prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/alerts/storage/cinder.yaml:42]: yes, those alerts are either relevant, we have `NetappHarvestExporterAuthenticationError` and `NetappHarvestExporterProbeError` in harvester

**Comment by chuan137** [comment]
yes, those alerts are not relevant, we have NetappHarvestExporterAuthenticationError and NetappHarvestExporterProbeError in harvester

**Review by Carthaca — APPROVED** [review]

**Pull request merged**